.env.baklavastaging

```diff
@@ -0,0 +1,107 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="staging"
+
+GETH_VERBOSITY=2
+
+VM_BASED=true
+
+KUBERNETES_CLUSTER_NAME="baklavastaging"
+KUBERNETES_CLUSTER_ZONE="us-west1-a"
+CLUSTER_DOMAIN_NAME="celo-testnet"
+
+TESTNET_PROJECT_NAME="celo-testnet"
+
+BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="c42d9c2f3ef5e79c152c44a82fa0f12233dcfb5c"
+BLOCKSCOUT_WEB_REPLICAS=3
+# Increment this value everytime you redeploy blockscout. Or else the deployment will fail due to the
+# existing database.
+BLOCKSCOUT_DB_SUFFIX="0"
+BLOCKSCOUT_SUBNETWORK_NAME="Baklava Staging"
+
+ETHSTATS_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/ethstats"
+ETHSTATS_DOCKER_IMAGE_TAG="bbc89603d872af1c6b37cf00e8220ed26237fc91"
+ETHSTATS_TRUSTED_ADDRESSES=""
+ETHSTATS_BANNED_ADDRESSES=""
+
+GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_NODE_DOCKER_IMAGE_TAG="a85db20179b503b802430aef7085d35aa761c63d"
+
+GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/geth-all"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="a85db20179b503b802430aef7085d35aa761c63d"
+
+CELOTOOL_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
+CELOTOOL_DOCKER_IMAGE_TAG="celotool-51ae8e72be34a614346950b6fbd8ba3b94223947"
+
+CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
+CELOCLI_STANDALONE_IMAGE_TAG="0.0.30-beta2"
+
+# Schedule for an oracle deployed via celotool, expressed in crontab syntax
+# This schedule is "every 5th minute"
+ORACLE_CRON_SCHEDULE="*/5 * * * *"
+
+ORACLE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/oracle"
+ORACLE_DOCKER_IMAGE_TAG="baklavastaging"
```

Comment:
The origin of this file is @tkporter's branch for baklavastaging deployment, as it is today (Dec 2). These vars are the only ones I added:

```
CELOCLI_STANDALONE_IMAGE_REPOSITORY
CELOCLI_STANDALONE_IMAGE_TAG
ORACLE_CRON_SCHEDULE
ORACLE_DOCKER_IMAGE_REPOSITORY
ORACLE_DOCKER_IMAGE_TAG
```

Reply:
The only one of these that should be different for `baklava` is:
`ORACLE_DOCKER_IMAGE_TAG="baklava"`

That docker image already exists and is ready to go.